### PR TITLE
feat(Gmail (46 > main)) create the reaction send mail

### LINKED ISSUE #46: Create the reaction send mail
Send a basic email with Gmail API

### PR DESCRIPTION
**Pull request by Pablodeibar**
Closes #46

### CODE REVIEW

**Review by Benjicatch — DISMISSED** [review]
ok

**Review by AlexGuillard — APPROVED** [review]
ok

File: server/src/mailing/mailing.service.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+    oauth2Client.setCredentials({
+      refresh_token: process.env.REFRESH_TOKEN,
+    });
+
+    const accessToken: string = await new Promise((resolve, reject) => {
+      oauth2Client.getAccessToken((err, token) => {
+        if (err) {
+          console.log(err);
+          console.log('GOOGLE_CLIENT_ID:', this.configService.get('GOOGLE_CLIENT_ID'));
```
Comment (Benjicatch) [inline server/src/mailing/mailing.service.ts:30]: remove console.log

**Review by Benjicatch — CHANGES_REQUESTED** [review]

**Review by AlexGuillard — APPROVED** [review]
ok

**Review by Benjicatch — APPROVED** [review]
ok

**Pull request merged**